superdesk/io/__init__.py

```diff
@@ -112,6 +112,18 @@ def set_dateline(self, item, city=None, text=None):
         if text:
             item['dateline']['text'] = text
 
+    def map_priority(self, source_priority):
+        """
+        Maps the source priority to superdesk priority
+        :param str source_priority:
+        :return int: priority of the item
+        """
+        if source_priority and source_priority.isdigit():
+            if int(source_priority) in [1, 2, 3]:
```

Comment:
Verified with Lasky, everything in the cyber system goes out as 5. 1-3 priority level are introduced in Superdesk and rest is 5. :smirk: